reportportal_client/helpers.py

```diff
@@ -106,3 +108,27 @@ def get_package_version(package_name):
     except DistributionNotFound:
         package_version = 'not found'
     return package_version
+
+
+def verify_value_length(attributes):
+    """Verify length of the attribute value.
+
+    The length of the attribute value should have size from '1' to '128'.
+    Otherwise HTTP response will return an error.
+    Example of the input list:
+    [{'key': 'tag_name', 'value': 'tag_value1'}, {'value': 'tag_value2'}]
+    :param attributes: List of attributes(tags)
+    :return:           List of attributes with corrected value length
+    """
+    if attributes is not None:
+        for pair in attributes:
+            if not isinstance(pair, dict):
+                continue
+            attr_value = pair.get('value')
+            if attr_value is None:
+                continue
+            try:
+                pair['value'] = attr_value[:ATTRIBUTE_LENGTH_LIMIT]
```

Comment:
why so complicated?
how about:
```
if len(attr_value) > ATTRIBUTE_LENGTH_LIMIT:
    raise TypeError(f"the length of the value of the attribute {pair.get('name')} exceeded the maximum allowed size ({ATTRIBUTE_LENGTH_LIMIT}))
```

Reply:
1. We should not raise anything except for connection issues. Otherwise, we will impact test sessions.
2. Your suggestion is in Python 3.

The idea is to reduce the length of the object if it supports slicing.